src/repl.coffee

```diff
@@ -49,7 +47,8 @@ replDefaults =
       isAsync = ast.isAsync
       # Invoke the wrapping closure.
       ast    = new Root new Block [new Call ast]
-      js     = ast.compile {bare: yes, locals: Object.keys(context), referencedVars, sharedScope: yes}
+      ast.findReferencedVars()
```

Comment:
Per my comment above, I’m wondering if we could do without this call, that it could happen automatically as part of assembling the nodes tree.